### DISCUSSION TITLE
Stop indexing with ongoing service agreements

### DISCUSSION THREAD

**Discussion by WayneWayner** [subquery/subql#961] [Frontier TestNet - Bug Reporting]
If you don't want to index a project anymore, you can press the stop indexing button. This will be performed immediately. But you can't send an announce not indexing tx, because there are still existing service agreements. 
It would be better if this information is displayed earlier during the stop indexing action.

**Comment by mzxyz** [comment]
Nice suggestion, added this to our backlog.

For our existing plan, we will remove the second step for stop indexing to make it a pure action to terminate the project. If there has ongoing service agreement, `announce not indexing` will be disabled with explanation.

Will also include the extra info if there has ongoing service agreements when stop indexing the project.
